assets/questions/isLeapYear.js

```diff
@@ -0,0 +1,196 @@
+        'If the year can be evenly divided by 100, it is NOT a leap year, ',
+        'unless:'
+      ].join(''),
+      type: 'text'
+    }, {
+      content: [
+        'The year is also evenly divided by 400. Then it is a leap year.'
+      ].join(''),
+      type: 'text'
+    }, {
+      content: [
+        'Write a function that takes a year and returns True if the year is a ',
+        'leap year, and False otherwise. For example:'
+      ].join(''),
+      type: 'text'
+    }, {
+      content: 'Input: 2018\nOutput: False',
+      type: 'code'
+    }],
+    prerequisiteSkills: [],
+    acquiredSkills: ['Conditionals'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'isLeapYear',
+    languageSpecificTips: {
+      python: []
+    },
+    testSuites: [{
+      id: 'NOT_DIVISIBLE_BY_FOUR',
+      humanReadableName: 'the not-divisible-by-4 case',
+      testCases: [{
+        input: 1903,
+        allowedOutputs: [false]
+      }, {
+        input: 1,
+        allowedOutputs: [false]
+      }, {
+        input: 1905,
+        allowedOutputs: [false]
+      }, {
+        input: 702,
+        allowedOutputs: [false]
+      }]
+    }, {
+      id: 'DIVISIBLE_BY_FOUR_BUT_NOT_HUNDRED',
+      humanReadableName: 'the divisible-by-4-but-not-100 case',
+      testCases: [{
+        input: 1904,
+        allowedOutputs: [true]
+      }, {
+        input: 1976,
+        allowedOutputs: [true]
+      }, {
+        input: 2004,
+        allowedOutputs: [true]
+      }, {
+        input: 2016,
+        allowedOutputs: [true]
+      }]
+    }, {
+      id: 'DIVISIBLE_BY_HUNDRED_BUT_NOT_FOUR_HUNDRED',
+      humanReadableName: 'the divisible-by-100-but-not-400 case',
+      testCases: [{
+        input: 1900,
+        allowedOutputs: [false]
+      }, {
+        input: 1700,
+        allowedOutputs: [false]
+      }, {
+        input: 2100,
+        allowedOutputs: [false]
+      }, {
+        input: 3400,
+        allowedOutputs: [false]
+      }]
+    }, {
+      id: 'DIVISIBLE_BY_FOUR_HUNDRED',
+      humanReadableName: 'the divisible-by-400 case',
+      testCases: [{
+        input: 2000,
+        allowedOutputs: [true]
+      }, {
+        input: 1600,
+        allowedOutputs: [true]
+      }, {
+        input: 400,
+        allowedOutputs: [true]
+      }, {
+        input: 1200,
+        allowedOutputs: [true]
+      }]
+    }],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.returnNone',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'It looks like your program is not returning anything. Did you ',
+          'forget to include a "return" statement?'
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.returnTrue',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          "It looks like you're consistently returning True for all test ",
+          "cases. This implies that every year is a leap year, but that's ",
```

Comment:
This implies that every year is a leap year, which is not true.

Reply:
Done.

(Also, I thought "but" was the issue here, but we seem to have preserved it below, so just wanted to check what the issue was. Was it the ellipses? They were originally there to signal to the student "now it's your turn to think and continue the chain of reasoning", but I'm fine with removing them.)